react-table/src/components/Table.js

```diff
@@ -152,14 +177,14 @@ export default function Table({ rows }) {
           <MuiTable>
             <TableHead
               numSelected={0}
-              // order={order}
-              // orderBy={orderBy}
+              order={sort.order}
+              orderBy={sort.sortBy}
               onSelectAllClick={handleSelectAllClick}
               onRequestSort={handleRequestSort}
-              rowCount={rows.length}
+              rowCount={displayRows().length}
             />
             <TableBody>
-              {rows.map((row) => {
+              {displayRows().map((row) => {
```

Comment:
should create at least constant with displayRows() value. (instead of twice `displayRows()`)

```typescript
const displayRows = displayRows() 
```